scss/mixins/_debug.scss

```diff
@@ -0,0 +1,81 @@
+//-----------------------------
+//
+// Debugging
+// =========
+//
+// Mixins for debugging class use, etc.
+//
+//-----------------------------
+
+// Simple toggle for switching debugging on and off
+$DBdebug: true;
+
+// Mixin that gives visual feedback in the DOM if a selector is used outside of
+// a given context. Useful for detecting misused Elements in BEM.
+//
+//  .simple {
+//    font-weight: bold;
+//  }
+//
+//    .simple__usage {
+//      color: red;
+//      @include nest-check(".simple");
+//    }
+//
+// Courtesy Harry Roberts (@csswizardry)
+@mixin nest-check($element) {
+    @if ($DBdebug == true) {
+        position: relative;
+        outline: 1px solid color(red);
```

Comment:
Property outline should be separated from the previous group of properties ending with position